doc/install/install-openshift.md

```diff
@@ -0,0 +1,214 @@
+once ready it will be marked `complete` while installing it will be marked `installing`
+
+#### Configure Istio
+
+```
+kubectl apply -f - <<EOF
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+  name: default
+spec:
+  version: v1.21.0
+  namespace: istio-system
+  # Disable autoscaling to reduce dev resources
+  values:
+    pilot:
+      autoscaleEnabled: false
+EOF
+```
+
+Wait for Istio to be ready
+
+```
+kubectl wait istio/default -n istio-system --for="condition=Ready=true"
+```
+
+
+### (Recommended) Thanos and Observability Stack
+
+Kuadrant provides a set of sample dashboards that use the known metrics exported by kuadrant and gateway components to provide insight into the different areas of your APIs and Gateways. While this is not essential it is recommended that you set up an observability stack. Below are links to the OpenShift docs on this and also a link to help with the setup of Thanos for metrics storage.
+
+OpenShift supports a user facing monitoring stack. This can be cofigured and setup via their documentation
+https://docs.openshift.com/container-platform/4.14/observability/monitoring/configuring-the-monitoring-stack.html
+
+- If you have user workload monitoring enabled. We Recommend configuring  remote write to a central storage system such as Thanos: 
+- [Remote Write Config](https://docs.openshift.com/container-platform/4.14/observability/monitoring/configuring-the-monitoring-stack.html#configuring_remote_write_storage_configuring-the-monitoring-stack)
+
+- [Kube Thanos](https://github.com/thanos-io/kube-thanos)
+
+- [Kuadrant Examples](https://docs.kuadrant.io/kuadrant-operator/doc/observability/examples/)
+
+
+### Install Kuadrant
+
+To install Kuadrant we will use the Kuadrant Operator. Before this lets set up some secrets we will need later.
+
+```
+
+kubectl create ns kuadrant-system
+
+```
+
+AWS Credential for TLS verification
+
+```
+kubectl -n kuadrant-system create secret generic aws-credentials \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+````  
+
+Redis Credential for shared counters
+
+```
+kubectl -n kuadrant-system create secret generic redis-config \
+  --from-literal=URL=$REDIS_URL  
+```  
+
+```
+kubectl create ns ingress-gateway
+```
+
+AWS Credential for managing DNS records
+
+```
+kubectl -n ingress-gateway create secret generic aws-credentials \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+```  
+
+To install the Kuadrant operator:      
+```
+kubectl apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: kuadrant-operator
+  namespace: kuadrant-system
+spec:
+  channel: preview
+  installPlanApproval: Automatic
+  name: kuadrant-operator
+  source: kuadrant-operator-catalog
```

Comment:
If we would like to direct users to use the released Kuadrant (from OperatorHub) we should use:
```
  source: community-operators
  sourceNamespace: openshift-operators
```

Reply:
good spot. This was left over from testing